Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -60,6 +60,10 @@ job-name-paramedic = paramedic
 job-name-guard = prison guard
 job-name-martialartist =  martial artist
 
+# Park
+job-name-sai = station AI
+job-name-css = chief service supervisor
+
```

Comment:
These should go in the SimpleStation FTL folder. 
Should be Resources/Locale/en-US/SimpleStation14/  if I'm not mistaken.